Create ui-c8y-1022-4-3-correct-filters-dropdown-max-height-9420-graft-release-cd-9453.md (#3491)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-4-3-correct-filters-dropdown-max-height-9420-graft-release-cd-9453.md

---
date: ""

> title: Data grid filters dropdown max-height corrected

Reviewer: ```suggestion
title: Fixed filter dropdown max-height in data grid component
```

> title: correct filters dropdown max-height (#9420) [GRAFT][release/cd] (#9453)

Reviewer 2: ```suggestion
title: Data grid filters dropdown max-height corrected
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62301
version: 1022.4.3
---

> Co-authored-by: Carlos Ceia <carlos.ceia@cumulocity.com>

Reviewer 2: ```suggestion
In the Data grid component the filters dropdown had an issue with its max-height, causing it to be cut off in certain scenarios. This has now been corrected by setting an appropriate max-height.
```

> In the Data grid component the filters dropdown had an issue with its max-height, causing it to be cut off in certain scenarios. This has now been corrected by setting an appropriate max-height.

Reviewer: ```suggestion
In the data grid component, the filter dropdown had an issue with its max-height, causing it to be cut off in certain scenarios. This issue has now been resolved by setting an appropriate max-height.
```
